lib/ansible/modules/cloud/docker/docker_image.py

```diff
@@ -521,7 +537,8 @@ def build_image(self):
             pull=self.pull,
             forcerm=self.rm,
             dockerfile=self.dockerfile,
-            decode=True
+            decode=True,
+            cache_from=self.cache_from
```

Comment:
I think this will still cause an error if a version before docker-py 2.1.0 is used, since the parameter still gets passed in to `self.client.build()`. It should probably be moved to a conditional below, where it only gets set if it's not `None`.